### PR TITLE
Fix issue 59

### LINKED ISSUE #59: Allow changing options of UI playground
Allow similar options to what UI GraphiQL allows

### PR DESCRIPTION
**Pull request by deinok**
Closes #59 

Also, this work for netcoreapp1.X and netcoreapp2.X

### CODE REVIEW

File: src/Ui.Playground/GraphiQLMiddleware.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+using System.Threading.Tasks;
+using GraphQL.Server.Ui.Playground.Internal;
+using Microsoft.AspNetCore.Http;
+
+namespace GraphQL.Server.Ui.Playground {
+
+    /// <summary>
+    /// A middleware for Playground
+    /// </summary>
+    public class PlaygroundMiddleware {
```
Comment (pekkah) [inline src/Ui.Playground/GraphiQLMiddleware.cs]: File name?

Comment (deinok) [reply src/Ui.Playground/GraphiQLMiddleware.cs]: Miss this :(
Fixed

File: src/Ui.Playground/Ui.Playground.csproj [hunk 2]
```diff
@@ -1,28 +1,31 @@
...
-  </ItemGroup>
+
+	<ItemGroup>
+		<EmbeddedResource Include="Internal\playground.cshtml" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.0.1" />
+		<PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="2.0.1" />
+		<PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" Version="2.0.0" />
```
Comment (pekkah) [inline src/Ui.Playground/Ui.Playground.csproj]: Some of these can now be removed as they're not required anymore.

Comment (deinok) [reply src/Ui.Playground/Ui.Playground.csproj]: Abstraction should stay. Its where HttpContext life

**Pull request merged**